### ISSUE TITLE
Firefox 35 has changed some things....

### ISSUE THREAD

**Issue by omgmog** [omgmog/install-all-firefox#54]
```
✔ Created profile '35' for Firefox 35.0
/tmp/firefoxes/install-all-firefox.sh: line 904: /Applications/Firefoxes/Firefox 35.0.app/Contents/MacOS/defaults/pref/all.js: No such file or directory
setfileicon: cannot load icon file
```

**Comment by omgmog** [comment]
Seems as if the ability to load extensions from `distribution/extensions` was removed.

**Comment by omgmog** [comment]
Okay the home for some things has been changed from:

`Firefox.app/Contents/MacOS/`

To

`Firefox.app/Resources/`

**Comment by omgmog** [comment]
Okay should be fixed since b2842275dacf2bd0e7eb9c954e193c5a9e093827 and f06002a7bf8c04423d19915cd750a508a0c09e69
